src/VirtoCommerce.CustomerReviews.Web/Module.cs

```diff
@@ -62,6 +68,19 @@ public void Initialize(IServiceCollection serviceCollection)
 
             serviceCollection.AddTransient<ReviewStatusChangedEventHandler>();
             serviceCollection.AddTransient<OrderChangedEventHandler>();
+
+            // GraphQL
+            var assemblyMarker = typeof(AssemblyMarker);
```

Comment:
Please move all these initializations to extensions method in  the VirtoCommerce.CustomerReviews.ExperienceApi.csproj

Reply:
Done